src/components/HeaderBar/SearchInput.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const Input = styled.input`
+  height: 40px;
+  font-size: 25px;
+  width: 100%;
+  border: 2px solid #aaa;
```

Comment:
I recommend handling the colors with a theme provider, so changing colors with the light and dark theme becomes easier:

https://styled-components.com/docs/advanced#function-themes